buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -87,6 +92,8 @@ class DependencyConstraints implements Plugin<Project> {
         api(group: 'cglib', name: 'cglib', version: get('cglib.version'))
         api(group: 'com.arakelian', name: 'java-jq', version: '1.1.0')
         api(group: 'com.carrotsearch.randomizedtesting', name: 'randomizedtesting-runner', version: '2.7.8')
+        api(group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-joda', version: '2.9.8')
+        api(group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.11.4')
```

Comment:
the current version on develop of datatype-joda and datatype-jsr310 is and should remain 2.12.1, as set down below around line 189, so please delete these two lines

Reply:
Fixed